bleachbit/FileUtilities.py

```diff
@@ -80,6 +80,36 @@ def open_files_linux():
     return glob.iglob("/proc/*/fd/*")
 
 
+def get_filesystem_type(path):
+    """
+    * Get file system type from the given path
+    """
+    try:
+        import psutil
+        
+        partitions = {}
+        for partition in psutil.disk_partitions():
+            partitions[partition.mountpoint] = (partition.fstype, partition.device)
+        
+        if path in partitions:
+            return partitions[path]
+
+        splitpath = path.split(os.sep)
+        for i in range(0, len(splitpath)-1):
+            path = os.sep.join(splitpath[:i]) + os.sep
+            if path in partitions:
+                return partitions[path]
+            
+            path = os.sep.join(splitpath[:i])
+            if path in partitions:
+                return partitions[path]
+
+        return ("unknown", "none")
+    except ImportError:
```

Comment:
How about movin the `except` right after `import` on line 88? It would be easier to read, and there's no way to throw an ImportError from lines 90-105.

Reply:
It is good idea. Let me modify it.